### DISCUSSION TITLE
Feature request: geomap color/size from query

### DISCUSSION THREAD

**Discussion by wz2b** [grafana/grafana#37756] [Dashboards, Panels & Visualizations]
I would like to have the geomap be able to set the size and/or color of a marker based on the content of the returned query.  Is this a feature others are interested in?

**Comment by nmarrs** [comment]
Hi @wz2b, Geomap now supports setting color / size of markers based on field data. See latest [geomap docs](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/geomap/) for more info

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
